src/libos/src/vm/vm_manager.rs

```diff
@@ -326,6 +326,11 @@ impl VMManager {
         self.mmap_prefered_start_addr = addr
     }
 
+    // This is used to clean all vmas
+    pub fn sanitize(&mut self) -> Result<()> {
+        self.munmap(self.range.start(), self.range().size())
```

Comment:
self.range.start() could be self.range().start()

Reply:
This part is useless. Removed.